Sources/Apollo/MultipartFormData.swift

```diff
@@ -34,7 +34,12 @@ public class MultipartFormData {
     self.init(boundary: "apollo-ios.boundary.\(UUID().uuidString)")
   }
 
-  func appendPart(string: String, name: String) throws {
+  /// Appends the passed-in string as a part of the body.
+  ///
+  /// - Parameters:
+  ///   - string: The string to append
+  ///   - name: The name of the part to pass along to the server
+  public func appendPart(string: String, name: String) throws {
```

Comment:
@designatednerd this was only internal. The `TestCustomRequestCreator` used it and I know I use in my project too. 
https://github.com/apollographql/apollo-ios/pull/804/files#diff-b1b6d9ae6149908bf9768bbc4bc4b3b0R50